scripts/styleChecker.fsx

```diff
@@ -0,0 +1,442 @@
+        else
+            raise <| ProcessFailed errMsg
+
+    | WarningsOrAmbiguous output ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = sprintf "%s (with warnings?)" errMsg
+        Console.Error.WriteLine fullErrMsg
+        fullErrMsg
+
+let IsProcessSuccessful(processResult: ProcessResult) : bool =
+    match processResult.Result with
+    | Success output -> true
+    | _ -> false
+
+let InstallPrettier(version: string) =
+    let isPrettierInstalled =
+        Process.Execute(
+            {
+                Command = "npm"
+                Arguments = $"list prettier@{version}"
+            },
+            Echo.All
+        )
+        |> IsProcessSuccessful
+
+    if not(isPrettierInstalled) then
+        Process.Execute(
+            {
+                Command = "npm"
+                Arguments = $"install prettier@{version}"
+            },
+            Echo.Off
+        )
+        |> UnwrapProcessResult None false
+        |> ignore
+
+let StyleFSharpFiles(rootDir: DirectoryInfo) =
+    InstallFantomlessTool fantomlessToolVersion
+
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments = $"fantomless --recurse {rootDir.FullName}"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+let StyleCSharpFiles(rootDir: DirectoryInfo) =
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments = $"format whitespace {rootDir.FullName} --folder"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+let InstallPrettierPluginXml(version: string) =
+    let isPrettierPluginXmlInstalled =
+        Process.Execute(
+            {
+                Command = "npm"
+                Arguments = $"list @prettier/plugin-xml@{version}"
+            },
+            Echo.Off
+        )
+        |> IsProcessSuccessful
+
+    if not(isPrettierPluginXmlInstalled) then
+        Process
+            .Execute(
+                {
+                    Command = "npm"
+                    Arguments = $"install @prettier/plugin-xml@{version}"
+                },
+                Echo.Off
+            )
+            .UnwrapDefault()
+        |> ignore
+
+let StyleXamlFiles() =
+    InstallPrettier prettierVersion
+    InstallPrettierPluginXml pluginXmlVersion
+
+    Process
+        .Execute(
+            {
+                Command = "npm"
+                Arguments =
+                    $"install --save-dev prettier@{prettierVersion} @prettier/plugin-xml@{pluginXmlVersion}"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+    let pattern = Path.Combine [| "**"; "*.xaml" |]
+
+    Process
+        .Execute(
+            {
+                Command =
+                    Path.Combine
```

Comment:
@realmarv I don't know why you started using Path.Combine as it receives an array, but that's not the way it should be used; Path.Combine receives arguments (C#-style arguments, which, to F# are shown as a tuple), even if the number of arguments is variable; so please just use parentheses when calling Path.Combine